File: 412/Solution.java
class Solution {
    public List<String> fizzBuzz(int n) {
       List<String> x = new ArrayList<String>();
       if(n == 0) return x;
       for(int i=1; i<n+1; i++) {
           if (i%15 == 0) {
               x.add("FizzBuzz");
           }
           else if(i%5 == 0) {
               x.add("Buzz");
           }
           else if (i%3 == 0) {
               x.add("Fizz");
           }
           else {
               x.add(i + "");
           }
       } 
        return x;
    }
}

File: 412/Solution.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        ls = list()
        if n == 0:
            return ls
        for i in range(1, n+1):
            if i%15 == 0:
                ls.append("FizzBuzz")
            elif i%5 == 0:
                ls.append("Buzz")
            elif i%3 == 0:
                ls.append("Fizz")
            else:
                ls.append(str(i))
        return ls
                